utility/mirrormanager2.spec

```diff
@@ -401,8 +398,7 @@ MM2_SKIP_NETWORK_TESTS=1 ./runtests.sh -v
 
 %files mirrorlist
 %config(noreplace) %{_sysconfdir}/httpd/conf.d/mirrorlist-server.conf
-%attr(755,mirrormanager,mirrormanager) %dir %{_localstatedir}/run/mirrormanager
-%attr(755,mirrormanager,mirrormanager) %dir %{_localstatedir}/lib/mirrormanager/
+%attr(755,root,root) %dir %{_localstatedir}/lib/mirrormanager/
```

Comment:
This are also unexpected changes.

Reply:
This was also rpmlint error.
Line 404 was `dir-or-file-in-var-run`. Here similar bugzilla report: https://bugzilla.redhat.com/show_bug.cgi?id=1438478
Line 405 was throwing some error too, but I'm unable to reproduce it anymore. Well, ok, if it does not complain now I can change it back to `mirrormanager` instead of `root` :)